File: spring-rest-api-galleria/src/main/java/com/progettogallery/dto/InsertImage.java
package com.progettogallery.dto;

public class InsertImage {
	private String titoloImg;
	private String titoloGalleria;
	private String urlImg;
	public String getTitoloImg() {
		return titoloImg;
	}
	public void setTitoloImg(String titoloImg) {
		this.titoloImg = titoloImg;
	}
	public String getTitoloGalleria() {
		return titoloGalleria;
	}
	public void setTitoloGalleria(String titoloGalleria) {
		this.titoloGalleria = titoloGalleria;
	}
	public InsertImage() {
		super();
	}
	public String getUrlImg() {
		return urlImg;
	}
	public void setUrlImg(String urlImg) {
		this.urlImg = urlImg;
	}
	
}

File: galleria-angular/src/app/galleria-service/InsertImage.ts
export class InsertImage {
    titoloImg !: string;
    titoloGalleria !: string;
    urlImmagine !:string;

    getTitoloImg():string {
        return this.titoloImg;
    }

     getTitoloGalleria():string {
        return this.titoloGalleria;
    }

    constructor(titoloImg: string, titoloG: string, urlImmagine:string){
        this.titoloImg = titoloImg;
        this.titoloGalleria = titoloG;
        this.urlImmagine = urlImmagine;
    }
}